bHY6yh-d29370/cod-klc89d_bHY6yh-d29370.xml

```diff
@@ -0,0 +1,414 @@
+              triplex difficile rumpitur</quote>
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e221">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e240">Plato in intimeo circa finem</ref> <quote xml:id="bHY6yh-d29370-Qd1e225">A bonis 
+              <cb ed="#Ka" n="b"/>
+              <lb ed="#Ka"/>docebris si nota malis commiscearis 
+              <lb ed="#Ka"/>perdes et inherentem intellectum</quote>
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e237">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e258">Socrates in exhortationibus capitulo 1</ref> // <quote xml:id="bHY6yh-d29370-Qd1e241">prauo<lb ed="#Ka" break="no"/>rum 
+              quidem consuetu dines paucum 
+              <lb ed="#Ka"/>tempus dissoluit studiosorum nota fedia 
+              <lb ed="#Ka"/>non quidem totum seculum abolenit</quote> //
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e251">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e272">capitulo 2</ref> <quote xml:id="bHY6yh-d29370-Qd1e253">placide detegoras ad omnes opti<lb ed="#Ka" break="no"/>mis 
+              autem vtere sic illis quidem odi<lb ed="#Ka" break="no"/>osus 
+              non eris hiis autem amicaberis 
+              <lb ed="#Ka"/>// Esto adherentibus tractabilis non 
+              <lb ed="#Ka"/>sublimis nam despiciendum tumorem 
+              <lb ed="#Ka"/>vix equidem tollerabunt serui modum 
+              <lb ed="#Ka"/>uero tractabilium cuncti fuerunt alacriter 
+              <lb ed="#Ka"/>fines autem tractabilis non eris rixosus 
+              <lb ed="#Ka"/>nec implacabilis nec apud eos litigi<lb ed="#Ka" break="no"/>osus 
+              nec adherentium iras aspere obuit<lb ed="#Ka" break="no"/>tus 
+              si iniuste irascantur Sed illis 
+              <lb ed="#Ka"/>quidem successientibus obtemperans 
+              <lb ed="#Ka"/>sed data uero ira obiurgans nec erga 
+              <lb ed="#Ka"/>ridicula studens nam omne in 
+              <lb ed="#Ka"/>tempestum vbique triste nec gratias ingrate 
+              <lb ed="#Ka"/>conferens quod nempe patiuntur 
+              <lb ed="#Ka"/>plures agentes ministrantes uero diletis 
+              <lb ed="#Ka"/>cum tedio ne amator eorum existens 
+              <lb ed="#Ka"/>molestum enim nec pronus percipere prouoca<lb ed="#Ka" break="no"/>tiuum 
+              enim // Conuersantes in domno<lb ed="#Ka" break="no"/>cratia 
+              oportet demulcere populum</quote>
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e291">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e316">Aristotiles ethicorum libro 8 capitulo 3</ref> <quote xml:id="bHY6yh-d29370-Qd1e295">Simile 
+              <lb ed="#Ka"/>simile appetit</quote> //
+            </p>
+            <p xml:id="bHY6yh-d29370-d1e301">
+              <ref xml:id="bHY6yh-d29370-Rd1e326">libro 9 capitulo 9</ref> <quote xml:id="bHY6yh-d29370-Qd1e303">Solitario 
```

Comment:
Notice the placement of the dbslash

```suggestion
            <p xml:id="bHY6yh-d29370-d1e291">
              <lb ed="#Ka"/><ref xml:id="bHY6yh-d29370-Rd1e316">Aristotiles ethicorum libro 8 capitulo 3</ref> <quote xml:id="bHY6yh-d29370-Qd1e295">Simile 
              <lb ed="#Ka"/>simile appetit</quote>
            </p>
            <p xml:id="bHY6yh-d29370-d1e301">
               // <ref xml:id="bHY6yh-d29370-Rd1e326">libro 9 capitulo 9</ref> <quote xml:id="bHY6yh-d29370-Qd1e303">Solitario 
```

Reply:
@kuriae can you take a look at these changes and accept them when you get a chance